newsroom/search.py

```diff
@@ -259,7 +259,43 @@ def get_internal_request(self, search):
         return internal_req
 
     def set_bool_query_from_filters(self, bool_query: Dict[str, Any], filters: Dict[str, Any]):
-        bool_query["must"] += [{"terms": {self.get_aggregation_field(key): val}} for key, val in filters.items() if val]
+        for key, val in filters.items():
+            if not val:
+                continue
+            nested_config = self.get_nested_config(key)
+            if nested_config:
```

Comment:
This will also need to be added for Agenda searches, https://github.com/superdesk/newsroom-core/blob/release/2.3/newsroom/agenda/agenda.py#L580